### PR TITLE
@cldougl Axes doc

### PR DESCRIPTION
**Pull request by ellecj**

### CODE REVIEW

File: _posts/plotly2/2016-11-28-axes-properties.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   sub-steps:
+    - copy: "You can also set up the axis type as linear or log. In the 'Range' sub-tab, select the x-axis or y-axis, then click 'Linear' or 'Log' under 'Axis Type'."
+      img: "![Linear log](../static/images/misc-tutorial-images/loglinear.gif)"
+
+ - title: Tick Labels
+   sub-steps:
+    - copy: "Customizing the tick labels along the axes can help highlight particular aspects of your data."
+    - copy: "The tick labels reflect column values that were selected in your trace(s)."
+      img: "![Tick labels](../static/images/misc-tutorial-images/x axis tick labels.png)"
+    - copy: "You have the option of showing these labels or hiding them. Go to 'Axes', then 'Tick Labels' and depending on the axis you want to change, select X or Y, or 'ALL'. The default is 'SHOW' but click 'Hide' if you don't wish to display them."
```
Comment (cldougl) [inline _posts/plotly2/2016-11-28-axes-properties.md]: X, Y or 'All'

File: _posts/plotly2/2016-11-28-axes-properties.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+ - title: Tick Text
+   sub-steps:
+    - copy: "When the tick labels are set to 'Show', you can change their typeface, font size and font color. Note that certain colors and typefaces are only available with a PRO subscription. Click [here](https://plot.ly/products/cloud/) to upgrade!"
+      img: "![Tick label attributes](../static/images/misc-tutorial-images/tick label attributes.png)"
+    - copy: "The ticks can be angled at 45°, 90°, 135°, or 180°. You can also add exponents, but it'll only work if your axes consist of numerical data."
+      img: "![Tick angels and exponents](../static/images/misc-tutorial-images/angle and exponents.gif)"
+    - copy: "You can add a prefix or suffix to your tick labels. If yours isn't listed, click on 'Custom' and type it in the field."
+    - copy: "For our example, our y-axis consists of data in Celcius, and our x-axis is in minutes. 'Minutes' isn't listed, so we'll select 'Custom' and type it in."
+      img: "![Label formatting](../static/images/misc-tutorial-images/label formatting.gif)"
```
Comment (cldougl) [inline _posts/plotly2/2016-11-28-axes-properties.md]: @ellecj thanks for adding a gif of this!! I remember the 1st time I selected that it seemed a bit strange cause you have to like click and type over custom (maybe just me ;)

File: _posts/plotly2/2016-11-28-axes-properties.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    - copy: "Once you've selected or entered a prefix or suffix, click the icon next to 'Label Positioning'."
+    - copy: "A modal will open with a few selections. You can place the prefix or suffix for the first or last tick label, all ticks, or none."
+      img: "![Label positioning](../static/images/misc-tutorial-images/label positioning.gif)"
+    
+ - title: Customize Number of Tick Markers
+   sub-steps:
+    - copy: "To set the number of ticks, go to 'Axes', then 'Tick Markers'. You have the option of setting them up as 'Linear' or Custom' in the same section."
```
Comment (cldougl) [inline _posts/plotly2/2016-11-28-axes-properties.md]: ahh this is a little confusing - you can select the number from either Tick Labels or Tick Markers 

additionally - we hide the marker on default so to get that section to appear you have to click inside/outside

File: _posts/plotly2/2016-11-28-axes-properties.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    - copy: "The plot below had the dates on the x-axis set as biannual. We'd like to show every consecutive year, so we select 'Custom' and enter the desired number of markers in the field."
+      img: "![Custom tick markers](../static/images/misc-tutorial-images/custom tick markers.png)"
+    - copy: "Choosing 'Linear' under 'Number of Markers' is where you add the unix timestamp to the step size and step offset fields. When you work with axis specifications, you have to convert the date to unix time in milliseconds for the step size. For more information, visit [this](http://help.plot.ly/make-a-time-series-graph/) page."
```
Comment (cldougl) [inline _posts/plotly2/2016-11-28-axes-properties.md]: woohoo no more unix timestamps
you can edit this section w/ the following info: https://plot.ly/javascript/reference/#layout-xaxis-dtick

File: _posts/plotly2/2016-11-28-axes-properties.md (hunk 3 above)
Comment (ellecj) [inline _posts/plotly2/2016-11-28-axes-properties.md]: I know, and I didn't know whether it was something that was going to change or if we do have both ways to select. :-/

File: _posts/plotly2/2016-11-28-axes-properties.md [hunk 5]
```diff
@@ -61,10 +61,11 @@ steps:
...
     - copy: "The plot below had the dates on the x-axis set as biannual. We'd like to show every consecutive year, so we select 'Custom' and enter the desired number of markers in the field."
       img: "![Custom tick markers](../static/images/misc-tutorial-images/custom tick markers.png)"
-    - copy: "Choosing 'Linear' under 'Number of Markers' is where you add the unix timestamp to the step size and step offset fields. When you work with axis specifications, you have to convert the date to unix time in milliseconds for the step size. For more information, visit [this](http://help.plot.ly/make-a-time-series-graph/) page."
+    - copy: "When you work with date or time series plots, you have to convert the date for the step size
+  and set offset. Let's say you set ticks on the 15th of every third month, set the 'Step Size' to '2000-01-15' and 'Step Offset' to 'M3'. To set ticks every 4 years, set it to 'M48'. Our plot display our data every year, so it's been set to 'M12'. For more information, visit [this](http://help.plot.ly/make-a-time-series-graph/) page."
```
Comment (cldougl) [inline _posts/plotly2/2016-11-28-axes-properties.md]: nice!

**Comment by cldougl** [comment]
looks great 💃 !

**Pull request merged**